nengo/spa/vocab.py

```diff
@@ -92,7 +92,7 @@ def create_pointer(self, attempts=100, unitary=False):
             count = 0
             p = pointer.SemanticPointer(self.dimensions, rng=self.rng)
             if self.vectors.shape[0] > 0:
-                while count < 100:
+                while count < attempts:
```

Comment:
Not that I know of.

Oh, and while I'm thinking about it, every now and then I realize that this attempt thing really should keep track of the best one that it finds, and use that if it can't find one smaller than max_similarity.  In other words, it could at least do the best it can, rather than just sticking with the last one if it fails.....